### PR TITLE
Fix doc and style of SEResNet

### PR DESCRIPTION
**Pull request by yuyu2172**

### CODE REVIEW

**Comment by yuyu2172** [comment]
@Hakuyume

File: docs/source/license.rst [hunk 1]
```diff
@@ -20,6 +20,8 @@ See the following resources for the terms of use of a model with weights pretrai
       - resource
     * - ResNet50/101/152 (:obj:`imagenet`)
       - * `ResNet50/101/152 (trained on ImageNet) <https://github.com/KaimingHe/deep-residual-networks#models>`_
+    * - SEResNet50/101/152 and SEResNeXt50/101 (:obj:`imagenet`)
+      - * `SEResNet50/101/152 and SEResNet50/101 (trained on ImageNet) <https://github.com/hujie-frank/SENet>`_
```
Comment (Hakuyume) [inline docs/source/license.rst]: How about separating them into two lines?

Comment (Hakuyume) [inline docs/source/license.rst]: `SEResNet50/101` -> `SEResNeXt50/101` (You forgot `X`)

**Comment by yuyu2172** [comment]
thanks

**Pull request merged**